src/main/java/listeners/VideoRecorder.java

```diff
@@ -0,0 +1,87 @@
+package listeners;
+
+import org.monte.media.Format;
+import org.monte.media.math.Rational;
+import org.monte.screenrecorder.ScreenRecorder;
+import webdriver.DriverManager;
+import ws.schild.jave.*;
+
+import java.awt.*;
+import java.io.File;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import static org.monte.media.FormatKeys.*;
+import static org.monte.media.VideoFormatKeys.*;
+
+public class VideoRecorder {
+    private final String RECORD_DIRECTORY = "videos\\";
+    private ScreenRecorder screenRecorder;
+    File newFileName = null;
+
+    public void startRecording() {
+        try {
+            GraphicsConfiguration gc = GraphicsEnvironment
+                    .getLocalGraphicsEnvironment().getDefaultScreenDevice()
+                    .getDefaultConfiguration();
+            File dir = new File(RECORD_DIRECTORY);
+            org.openqa.selenium.Point point = DriverManager.getDriver().manage().window().getPosition();
+            org.openqa.selenium.Dimension dimension = DriverManager.getDriver().manage().window().getSize();
+            Rectangle rectangle = new Rectangle(point.x, point.y,
+                    dimension.width, dimension.height);
+            this.screenRecorder = new ScreenRecorder(gc, rectangle,
+                    new Format(MediaTypeKey, MediaType.FILE, MimeTypeKey,
+                            MIME_AVI),
+                    new Format(MediaTypeKey, MediaType.VIDEO, EncodingKey,
+                            ENCODING_AVI_TECHSMITH_SCREEN_CAPTURE,
+                            CompressorNameKey,
+                            ENCODING_AVI_TECHSMITH_SCREEN_CAPTURE, DepthKey,
+                            24, FrameRateKey, Rational.valueOf(15), QualityKey,
+                            1.0f, KeyFrameIntervalKey, 15 * 60), new Format(
+                    MediaTypeKey, MediaType.VIDEO, EncodingKey,
+                    "black", FrameRateKey, Rational.valueOf(30)), null,
+                    dir);
+            this.screenRecorder.start();
+        } catch (Exception e) {
+            System.out.println(e);
```

Comment:
why do you use sout instead of logging? 
Why part of your logged messages are done by sout and part by logger?
framework MUST have the SAME style for EVERYTHING.